src/DurableTask.Netherite/StorageLayer/Faster/StoreWorker.cs

```diff
@@ -561,6 +658,10 @@ public async Task<(long,(long,int))> WaitForCheckpointAsync(bool isIndexCheckpoi
 
                 this.partition.Settings.TestHooks?.CheckpointInjector?.CompactionComplete(this.partition.ErrorHandler);
             }
+            else
+            {
+                this.traceHelper.FasterProgress($"RunCompactionAsync: skipped due to 'null' target");
```

Comment:
Is there anything more we can say about the meaning of a "null" target? I don't expect a customer or a CSS engineer will understand what this means.

Reply:
I've refactored this method so that this log can be more informative. Please see the latest iteration